src/main/java/com/faforever/client/game/GamesTilesContainerController.java

```diff
@@ -27,14 +35,19 @@
 
   private final UiService uiService;
   private final ListChangeListener<Game> gameListChangeListener;
+  private final PreferencesService preferencesService;
   public FlowPane tiledFlowPane;
   public ScrollPane tiledScrollPane;
-  private Map<Integer, Node> uidToGameCard;
+  @VisibleForTesting
+  protected Map<Integer, Node> uidToGameCard;
```

Comment:
Should be package private, not protected.

Reply:
But then it is no longer VisibleForTesting!!!!

Reply:
Yes it is. I said `package private` not `private`. Means, no modifier, means, the field will be visible from the same package. And unit tests are in the same package :)